src/main/java/org/prebid/server/auction/CurrencyConversionService.java

```diff
@@ -0,0 +1,198 @@
+package org.prebid.server.auction;
```

Comment:
Since we have `org.prebid.server.currency` package now it seems consistent to move this service there.